scanner/src/styles/variables.scss

```diff
@@ -0,0 +1,2 @@
+$color-primary-dark: #20303e;
```

Comment:
Just wonder why these names were chosen for color tokens names? I thought it's colors from our palette, but didn't find em there. Maybe it could be called smth like just `color-gray` and `color-dark-gray`? Just thinking aloud

Reply:
They seem to be the same names as in our other codebases

Reply:
That's what I meant. It's IOXIO colors in React and Svelte libs. And I thought that's a reference to them

![Screenshot from 2024-03-27 18-49-59](https://github.com/ioxiocom/ioxio-tags/assets/73995378/97c56fa2-8368-4fee-9250-6b51374a20be)
![Screenshot from 2024-03-27 18-48-59](https://github.com/ioxiocom/ioxio-tags/assets/73995378/3aa80fc1-2eb5-4581-b68d-cc7c5f22740a)

Reply:
it's "primary" "dark", not "ioxio-primary-dark"